starters/next-wordpress-starter/next.config.js

```diff
@@ -38,6 +38,20 @@ if (process.env.WPGRAPHQL_URL === undefined) {
 // remove trailing slash if it exists
 imageDomain = imageDomain.replace(/\/$/, '');
 
+let PANTHEON_ENVIRONMENT_PREFIX, IS_LIVE_ENVIRONMENT;
```

Comment:
Use `PANTHEON_ENVIRONMENT`
```suggestion
let PANTHEON_ENVIRONMENT, IS_LIVE_ENVIRONMENT;
```

Reply:
Updated, also removed a duplicate line I had left in from tested. 👍